modules/apigee_edge_teams/src/Controller/TeamMembersList.php

```diff
@@ -116,6 +116,7 @@ public function overview(TeamInterface $team) {
     if (!empty($members)) {
       $user_storage = $this->entityTypeManager()->getStorage('user');
       $uids = $user_storage->getQuery()
+        ->accessCheck(FALSE)
```

Comment:
Personally, I'd require a comment above _every_ `->accessCheck(FALSE)` that explains clearly why the code author thought once that disabling access checking on the query level is necessary and does not lead to an information disclosure vulnerability or worse... because this is what is being risked every time.

Reply:
Comments added.